packages/flutter/lib/src/material/carousel.dart

```diff
@@ -1535,6 +1535,62 @@ class CarouselController extends ScrollController {
     }
   }
 
+  /// Animates the controlled carousel to the given item index.
+  ///
+  /// For [CarouselView], this will scroll the carousel so the item at [index] becomes
+  /// the leading item.
+  ///
+  /// If the [index] is less than 0, the carousel will scroll to the first item.
+  /// If the [index] is greater than the number of items, the carousel will scroll
+  /// to the last item.
+  /// For [CarouselView.weighted], this will scroll the carousel so the first item with
+  /// maximum weight becomes the leading item, with [index] determining the final layout
+  /// position. With `consumeMaxWeight` set to true, the item with the maximum weight
+  /// will be the leading item.
```

Comment:
```suggestion
  /// position.
```
I don't think we need this because the result should look the same whether `consumeMaxWeight` is true or false.